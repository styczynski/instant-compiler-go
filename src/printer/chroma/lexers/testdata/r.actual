items %>%
	group_by(id) %>%
	filter(n() > 2) %>%
	mutate(count = n()) %>%
	ungroup() %>%
	distinct(id, .keep_all = TRUE)
	write_csv("items.csv")
