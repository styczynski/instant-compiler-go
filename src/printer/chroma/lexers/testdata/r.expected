[
  {"type":"Name","value":"items"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"%\u003e%"},
  {"type":"Text","value":"\n\t"},
  {"type":"NameFunction","value":"group_by"},
  {"type":"Punctuation","value":"("},
  {"type":"Name","value":"id"},
  {"type":"Punctuation","value":")"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"%\u003e%"},
  {"type":"Text","value":"\n\t"},
  {"type":"NameFunction","value":"filter"},
  {"type":"Punctuation","value":"("},
  {"type":"NameFunction","value":"n"},
  {"type":"Punctuation","value":"()"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"\u003e"},
  {"type":"Text","value":" "},
  {"type":"LiteralNumber","value":"2"},
  {"type":"Punctuation","value":")"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"%\u003e%"},
  {"type":"Text","value":"\n\t"},
  {"type":"NameFunction","value":"mutate"},
  {"type":"Punctuation","value":"("},
  {"type":"Name","value":"count"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":" "},
  {"type":"NameFunction","value":"n"},
  {"type":"Punctuation","value":"())"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"%\u003e%"},
  {"type":"Text","value":"\n\t"},
  {"type":"NameFunction","value":"ungroup"},
  {"type":"Punctuation","value":"()"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"%\u003e%"},
  {"type":"Text","value":"\n\t"},
  {"type":"NameFunction","value":"distinct"},
  {"type":"Punctuation","value":"("},
  {"type":"Name","value":"id"},
  {"type":"Punctuation","value":","},
  {"type":"Text","value":" "},
  {"type":"Name","value":".keep_all"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":" "},
  {"type":"KeywordConstant","value":"TRUE"},
  {"type":"Punctuation","value":")"},
  {"type":"Text","value":"\n\t"},
  {"type":"NameFunction","value":"write_csv"},
  {"type":"Punctuation","value":"("},
  {"type":"LiteralString","value":"\"items.csv\""},
  {"type":"Punctuation","value":")"},
  {"type":"Text","value":"\n"}
]
